edx_api/user_info/__init__.py

```diff
@@ -36,3 +36,34 @@ def get_user_info(self):
         resp.raise_for_status()
 
         return Info(resp.json())
+
+    def update_user_name(self, username, full_name):
+        """
+        Updates full name of the user
+
+        Args:
+            username (user.models.User): Username of the Application user
+            full_name: Full name that will replace the user's existing full name
```

Comment:
Just a couple more docstring changes

```suggestion
            username (str): Username of the Application user
            full_name (str): Full name that will replace the user's existing full name
```